app/assets/javascripts/oxalis/api/api_latest.js

```diff
@@ -229,6 +230,32 @@ class TracingApi {
     }
   }
 
+  //  SKELETONTRACING API
+
+  /**
+   * Increases the node radius of the active node by multiplying it with 1.05^delta.
+   *
+   * @example
+   * api.tracing.setRadius(1)
+   */
+  setRadius(delta: number): void {
+    getActiveNode(Store.getState().tracing)
```

Comment:
As I already mentioned in our meeting on Weds, I'd like the API to be more generic, i.e. allow to specify specific nodes wherever possible and default to activeNode. The same for edges, trees etc.

Something like:
```
setNodeRadiusByDelta(delta: number, nodeId?: number, treeId?: number): void {
  getNodeAndTree(Store.getState().tracing, nodeId, treeId).
    map(([tree, node]) => Store.dispatch(setNodeRadiusAction(node.id, tree.id, node.radius * Math.pow(1.05, delta)))));
```